ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -54,16 +54,26 @@ def selu(input_x):
 def softsign(x):
     return ivy.divide(x, ivy.add(1, ivy.abs(x)))
 
+
 @with_supported_dtypes(
-    {"2.0.0 and below": ("int8", "int16", "int32", "int64", "float16", "float32", "float64")},
+    {
+        "2.0.0 and below": (
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "float16",
+            "float32",
+            "float64",
+        )
+    },
```

Comment:
lets leave out these changes, not really necessary

Reply:
@Sam-Armstrong these are lint errors, when i was running pre-commit all these changes are coming